stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux

```diff
@@ -0,0 +1,105 @@
+//Final working code as of August 11, 2020
+//Currently supports single field classification and binary data sets 
+
+import "system"
+
+naiveBayes = (myClass, myBucket, myField, myMeasurement) => {
```

Comment:
Oh this is so cool to see in Flux!! Reminds me of the DS classes that I took. Mad props for working this out in Flux!